account/user_blackbox_test.go

```diff
@@ -95,7 +95,8 @@ func (s *userBlackBoxTest) TestExistsUser() {
 		// Check not existing
 		err := s.repo.CheckExists(s.ctx, uuid.NewV4().String())
 		// then
-		require.IsType(s.T(), &goa.ErrorResponse{}, err)
+		var errorType errors.NotFoundError
+		require.IsType(s.T(), errorType, err)
```

Comment:
I ended it up having this funny issue yesterday `Object expected to be of type errors.NotFoundError, but was errors.NotFoundError`. So I worked it around with the current code. However, I found what was the problem with a bit of more time this morning.